src/core/ddsi/src/q_xevent.c

```diff
@@ -581,11 +581,10 @@ static void handle_xevk_entityid (struct nn_xpack *xp, struct xevent_nt *ev)
 #ifdef DDSI_INCLUDE_SECURITY
 static void send_heartbeat_to_all_readers(struct nn_xpack *xp, struct xevent *ev, struct writer *wr, nn_mtime_t tnow)
 {
-  struct nn_xmsg **msg = NULL;
   struct whc_state whcst;
   nn_mtime_t t_next;
   int hbansreq = 0;
-  unsigned count = 0, n = 0, i;
+  unsigned count = 0;
```

Comment:
``count`` is no longer actually used — I am guessing you forgot to remove it but perhaps you have a some other plan with it?